device/telescope.py

```diff
@@ -207,7 +209,7 @@ def on_put(self, req: Request, resp: Response, devnum: int):
                 seestar_dev[devnum].end_watch_thread()
             resp.text = MethodResponse(req).json
         except Exception as ex:
-            resp.text = MethodResponse(req, DriverException(0x500, 'Telescope.Connected failed', ex)).json
+            resp.text = MethodResponse(req, DevDriverException(0x500, 'Telescope.Connected failed', ex)).json
```

Comment:
@YungKC - This change, and the ones below it in this file require your review when you have time.
The `DriverException` class from `alpaca.telescope` does not include the third `exc` exception parameter.
However, the `DevDriverException` does. I presume this was the one intended, but please take a look to confirm.